packages/dataviews/src/dataforms-layouts/panel/index.tsx

```diff
@@ -138,7 +157,13 @@ function PanelDropdown< Item >( {
 					) }
 					onClick={ onToggle }
 				>
-					<fieldDefinition.render item={ data } />
+					{ showMixedValue ? (
+						__( 'Mixed' )
```

Comment:
In one of the issues, @jasmussen mentions the possibility to render a different "mixed" label/component based on the "type" of the control. So potentially we could defer this to the control/field itself at some point.

Reply:
The context is loosely discussed [here](https://github.com/WordPress/gutenberg/issues/55101#issuecomment-2527826801).